server/repository/src/db_diesel/consumption.rs

```diff
@@ -11,7 +11,7 @@ table! {
         id -> Text,
         item_id -> Text,
         store_id -> Text,
-        quantity -> BigInt,
```

Comment:
Note, this isn't a problem for Sqlite but luckly Postgres tests were failing otherwise I would have missed this one. This change came indirectly from changing the view.